dockerfiles/config/config-docker.ttl

```diff
@@ -0,0 +1,146 @@
+@prefix void: <http://rdfs.org/ns/void#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix wv: <http://vocab.org/waiver/terms/norms> .
+@prefix sd: <http://www.w3.org/ns/sparql-service-description#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix skosmos: <http://purl.org/net/skosmos#> .
+@prefix isothes: <http://purl.org/iso25964/skos-thes#> .
+@prefix mdrtype: <http://publications.europa.eu/resource/authority/dataset-type/> .
+@prefix : <#> .
+
+# Skosmos main configuration
+
+:config a skosmos:Configuration ;
+    # SPARQL endpoint
+    # a local Fuseki server is usually on localhost:3030
+    #skosmos:sparqlEndpoint <http://localhost:3030/ds/sparql> ;
+    # use the dev.finto.fi endpoint where the example vocabularies reside
+    skosmos:sparqlEndpoint <http://api.dev.finto.fi/sparql> ;
```

Comment:
Same here as below...could this be changed to `localhost:3030` or something?

Reply:
The issue here is that this `config-docker.ttl` is used when you run only the Skosmos docker container, without a Fuseki endpoint. That means that by default anyone running this image would get an empty/blank Skosmos page, and have to understand that they need to re-create the configuration with the right endpoint… or mount a volume overwriting the `config.ttl`…

If we set to `localhost:3030`, then by default every container will show errors (in the console if enabled I think?) unless the user has a) a fuseki endpoint running and b) it is mapped to the Docker container so it can access it as localhost:3030 (I think it would have to be <fuseki-container-name>:3030 or some Docker-networking-fu trick; though not sure if using localhost inside the skosmos-web container would work here).

WDYT?

Reply:
I've used `localhost:3030`, but had to change the command to use `--net=host`. That effectively means that the container started with `docker run ...` has access to the host network, and can access `localhost:3030`, which can be another container, a `jvm` process, etc.

I've also added notes telling users they can customize it, other ways to achieve the same without Docker Compose, like creating a network. I was going to write the complete example creating a network, but that would become quite extensive.